### ISSUE TITLE
Error on run start

### ISSUE THREAD

**Issue by julienmarantes** [asual/mxgraph-factory#1]
I just installed your awesome package in an Angular 7 project and trying to make it work.

I get this error at start up in a component : 

```
Uncaught ReferenceError: mxForceIncludes is not defined
    at eval (mxClient.js:407)
    at Object../node_modules/mxgraph/javascript/mxClient.js (mxgraph-factory.js:97)
    at __webpack_require__ (mxgraph-factory.js:21)
    at exports.mxgraphFactory (mxgraph-factory.ts:7)
    at Module../src/app/mxgraph/components/mxgraph.component.ts (mxgraph.component.ts:5)
    at __webpack_require__ (bootstrap:79)
    at Module../src/app/mxgraph/mxgraph.module.ts (mxgraph.component.ts:15)
    at __webpack_require__ (bootstrap:79)
    at Module../src/app/app.module.ts (app.component.ts:10)
    at __webpack_require__ (bootstrap:79)
```
At this line `at Module../src/app/mxgraph/components/mxgraph.component.ts (mxgraph.component.ts:5)` : 

`const { mxGraph, mxGraphModel } = mxgraphFactory({`

I literraly copied your example

EDIT 1 -----------------

Changed those lines : 

```
const { mxGraph, mxGraphModel } = mxgraphFactory({
  mxLoadResources: false,
  mxLoadStylesheets: false,
  mxForceIncludes: false,  //Added
  mxResourceExtension: '',  //Added
});

Resolve the error but I don't know why I have to since it's not specified
```

**Comment by asual** [comment]
Are you running into this using published 3.9.8 version or the latest code in the master branch?

**Comment by julienmarantes** [comment]
I just installed it from npm without specifying the version

So I'm running `3.9.8`
